src/Compatibility/Core/src/Android/Resources/values/styles.xml

```diff
@@ -24,29 +24,4 @@
     <item name="scrollViewStyle">@style/scrollViewScrollBars</item>
   </style>
 
-  <style name="MainTheme" parent="MainTheme.Base">
-  </style>
-  
-  <!-- Base theme applied no matter what API -->
-  <style name="MainTheme.Base" parent="Theme.AppCompat.DayNight.DarkActionBar">
```

Comment:
Equivalents of these moved to `src/Core/src/Platform/Android/Resources/values/styles.xml`